try:
    print("3" + 3)
except Exception as r:
    print(r)
